kaolin/graphics/SoftRenderer.py

```diff
@@ -495,11 +533,16 @@ def look(self, vertices, eye, direction=torch.FloatTensor([0, 1, 0]),
 
         return vertices
 
-    def perspective_distort(self, vertices, angle=30.):
+    def perspective_distortion(self, vertices, angle=30.):
         r"""Compute perspective distortion from a given viewing angle.
+
+        Returns:
+            vertices (torch.Tensor): Input vertices transformed to the camera coordinate
+                frame (shape: :math:`B \times V \times 3`), where :math:`B` is the batchsize,
+                and :math:`V` is the number of vertices in the mesh.
         """
         device = vertices.device
-        angle = torch.FloatTensor([angle * 180 / kal.math.pi]).to(device)
+        angle = torch.FloatTensor([angle * 180 / torch.FloatTensor([math.pi])]).to(device)
```

Comment:
Avoid using `torch.FloatTensor([math.pi])` and directly use `math.pi`

Reply:
Had to do it this way, or it was complaining (idk why).

Reply:
On my workstation, using pytorch 1.2
```>>> torch.FloatTensor([1 * 180 / math.pi]).to('cuda')
tensor([57.2958], device='cuda:0')
```
What kind of error did you have ?

Reply:
Complaining about being unable to create the tensor on 'cpu'. I'm using pytorch 1.4

Reply:
on pytorch 1.4 (docker images: pytorch/pytorch:1.4-cuda10.0-cudnn7-devel):
```
>>> torch.FloatTensor([1 * 180 / math.pi]).to('cpu')
tensor([57.2958])
>>> torch.FloatTensor([1 * 180 / math.pi]).to('cuda')
tensor([57.2958], device='cuda:0')
```
I'm suspecting something else going wrong